gui/storage/models.py

```diff
@@ -247,9 +247,13 @@ def has_attachments(self):
             for vm_attached in vms_attached:
                 attachments['vm'].append(vm_attached.get('device_id'))
 
+        jails_licensed = False
         with client as c:
-            attachments['jails_ng'] = [
-                ij['host_hostuuid'] for ij in c.call('jail.query')]
+            jails_licensed = c.call('system.feature_enabled', 'JAILS')
+        if jails_licensed:
```

Comment:
Can we put this inside the first with statement so we dont have to open two connections in case jails is enabled?

Reply:
Sure!